shopinvader/models/shopinvader_backend.py

```diff
@@ -71,6 +71,15 @@ class ShopinvaderBackend(models.Model):
         help="If checked, use the specific shopinvader display name for "
              "products instead of the original product name."
     )
+    category_binding_level = fields.Integer(
+        default=3,
```

Comment:
A value to 0 is the only way to inactivated the feature, I would set the default = 0